### ISSUE TITLE
How to implement PHPnuke

### ISSUE THREAD

**Issue by narcisgarcia** [phpnuke/phpnuke#3]
It's needed to provide a step-by-step guide of how-to download and install PHPnuke on a LAMP server.

**Comment by meis2m** [comment]
> It's needed to provide a step-by-step guide of how-to download and install PHPnuke on a LAMP server.

Hi, it is easy download from here
https://github.com/phpnuke/phpnuke/archive/refs/tags/v8.4.4.zip

and install it like wordpress and other cmses

**Comment by narcisgarcia** [comment]
Now it's time to edit project's README.md and link phrase "Get your copy of phpNuke" to this answered URL.
And write a step-by-step guide "like wordpress and other CMSes" to not depend on other CMSes documentation.

**Comment by meis2m** [comment]
> Now it's time to edit project's README.md and link phrase "Get your copy of phpNuke" to this answered URL. And write a step-by-step guide "like wordpress and other CMSes" to not depend on other CMSes documentation.

u right we should prepare a good doc, en language added recently and need some more time to do stuffs

**Comment by narcisgarcia** [comment]
Thank you for enerything in this path, and thank you for download link.

**Comment by narcisgarcia** [comment]
Warning: chmod(): Operation not permitted in /srv/www/site.tree/public/includes/class.cache.php on line 349

Fatal error: Uncaught Exception: cache/ must be readable and writeable in /srv/www/site.tree/public/includes/class.cache.php:350 Stack trace: #0 /srv/www/site.tree/public/includes/class.cache.php(303): Cache->_checkCacheDir() #1 /srv/www/site.tree/public/includes/class.cache.php(264): Cache->getCacheDir() #2 /srv/www/site.tree/public/includes/class.cache.php(98): Cache->_loadCache() #3 /srv/www/site.tree/public/install.php(112): Cache->isCached('install_options') #4 {main} thrown in /srv/www/site.tree/public/includes/class.cache.php on line 350

**Comment by narcisgarcia** [comment]
Strict CSP-SOP does not allow inline JavaScript, this means the "onchange" event for selectlang option control does not work to switch to english.
A manual ?install_lang=english must be appended to install.php URL.

**Comment by narcisgarcia** [comment]
Although I've set apparently required permissions:
```
chmod ug+rw config.php
chmod -R ug+rwX cache files/uploads files/Articles
```
Web installer says:
```
Check server requirements
PHP Version: 	7.3.31-1~deb10u1
Supported Databeses: 	mysql, sqlite
Supported translation plugins 	Multi-Byte, iconv
config.php Permissions: 	OK
cache Dir Permissions: 	No permission
files/Uploads/ Dir Permissions: 	No permission
files/Articles/ Dir Permissions: 	No permission

Error

To install PHPNuke 8.4.4, you must perform the following modifications, then install the PHPNuke
The cache directory (cache/) is not rewritable. Please make 777 access to this directory
The uploads directory (files/uploads/) is not rewritable. Please make 777 access to this directory
The uploads directory (files/Articles/) is not rewritable. Please make 777 access to this directory
```
Permissions check fail unless rw is set to their parent directories too:
```
chmod ug+rwX .
chmod ug+rwX files
```

**Comment by narcisgarcia** [comment]
During installation web process I specified database password and it not asked me for first account password (only user name).
Then I don't know what's the default or first password for first custom account.

I've tried that pencil icon (Forget/Reset Password) and reached "Email Reset Linke Was Sent To Yur Email", but validation link returns this in web page:
`Bad Reset Code`

I've also tried by adding this directive to website's Virtualhost:
`php_admin_value request_order "GPC"`
but no luck.

What to do now?
